index.php

```diff
@@ -0,0 +1,36 @@
+<?php
+require_once "Autor.php";
+require_once "Libro.php";
+require_once "Revista.php";
+require_once "ImprimirAutor.php";
+
+
+$autorElenaWhite = new Autor("Elena G. de White", "Americana");
+$autorCSLewis    = new Autor("C. S. Lewis", "Británico");
+$autorGabo       = new Autor("Gabriel García Márquez", "Colombiano");
+$autorBorges     = new Autor("Jorge Luis Borges", "Argentino");
+
+
+$imp = new ImprimirAutor();
```

Comment:
```suggestion
$impresor = new ImprimirAutor();
```
Usar un nombre más descriptivo